starter-code/src/functions-and-arrays.js

```diff
@@ -1,4 +1,14 @@
 // Find the maximum
+function maxOfTwoNumbers(numberOne, numberTwo) {
```

Comment:
You can either create three conditions (equal, greater or smaller) or only two with (greater or equal and smaller or the opposite)